specs/data_structures.md

```diff
@@ -385,9 +386,9 @@ Note that while data is laid out in a two-dimensional square, rows and columns a
 
 Reed-Solomon erasure coding is used as the underlying coding scheme. The parameters are:
 - 16-bit Galois field
-- `AVAILABLE_DATA_ORIGINAL_SQUARE_SIZE` original pieces
-- `AVAILABLE_DATA_ORIGINAL_SQUARE_SIZE` parity pieces (i.e `AVAILABLE_DATA_ORIGINAL_SQUARE_SIZE * 2` total pieces), for an erasure efficiency of 50%. In other words, any 50% of the pieces from the `AVAILABLE_DATA_ORIGINAL_SQUARE_SIZE * 2` total pieces are enough to recover the original data.
-- `SHARE_SIZE` bytes per piece
+- [`availableDataOriginalSquareSize`](#header) original pieces (maximum of [`AVAILABLE_DATA_ORIGINAL_SQUARE_MAX`](./consensus.md#constants))
```

Comment:
is it the number of shares or the number rows/cols as mentioned above?

Reply:
Both? If there are X rows, then a column has X shares by construction.

Reply:
If a column has X shares, shouldn't the total number of original pieces (shares) should be X times X and not only X?

Reply:
Each erasure coding operation happens on a single row or column, not on the entire square.

Reply:
OK, so this doesn't refer to the total number of original pieces but the number of original pieces per row (or column). I think this becomes clearer where we talk about the 2D scheme.